src/library/simplified/extension/cabin-class.js

```diff
@@ -3,9 +3,9 @@ if (utag.isCruiseCO() && b['entity.checkout.cruise.cabinCategory'])
 {
     b['cabinClass'] = b['entity.checkout.cruise.cabinCategory'];
 }
-else if (utag.isFRateDetails() && b['entity.tripDetails.flightOffer.flight.legs.0.segments.0.cabinClass'])
+else if (utag.isFRateDetails() && b['entity.flight.flight.flight.legs.0.segments.0.cabinClass'])
 {
-    b['cabinClass'] = b['entity.tripDetails.flightOffer.flight.legs.0.segments.0.cabinClass'];
+    b['cabinClass'] = b['entity.flight.flight.flight.legs.0.segments.0.cabinClass'];
```

Comment:
it will be available with this PR: https://swarm-1994.sea.corp.expecn.com/reviews/1759862